stl/inc/xmemory

```diff
@@ -73,12 +73,13 @@ _INLINE_VAR constexpr size_t _New_alignof = (_STD max)(alignof(_Ty),
 
 // STRUCT _Default_allocate_traits
 struct _Default_allocate_traits {
-    __declspec(allocator) static void* _Allocate(const size_t _Bytes) {
+    __declspec(allocator) static _CONSTEXPR20_DYNALLOC void* _Allocate(const size_t _Bytes) {

```

Comment:
As shown in the commits on my subbranch, this can (but must) be constexpr for clang, as MSVC only allows std::allocator<T>::allocate

Reply:
Resolved from 35e4aeb